hassio/src/dashboard/hassio-update.ts

```diff
@@ -159,10 +159,18 @@ export class HassioUpdate extends LitElement {
       return;
     }
     try {
+      // Store the current version in local storage
+      if (item.name === "Home Assistant Core") {
+        localStorage.PendingCoreUpgrade = this.hass.config.version;
+      }
+
       await this.hass.callApi<HassioResponse<void>>("POST", item.apiPath);
     } catch (err) {
-      // Only show an error if the status code was not 504 (timeout reported by proxies)
-      if (err.status_code !== 504) {
+      // Only show an error if the status code was not 504 (timeout reported by proxies) and if the WS is connected
+      if (
+        err.status_code !== 504 &&
+        this.hass.connection.socket.readyState === 1
```

Comment:
This doesnt work?
```suggestion
        this.hass.connected
```

Reply:
Was hoping it did and committed that first.
But that stays `true`:/

Reply:
Found a better way